slack_sdk/socket_mode/aiohttp/__init__.py

```diff
@@ -250,38 +253,41 @@ async def receive_messages(self) -> None:
-                                f"extra: {message.extra}, "
-                                f"session: {session_id})"
+                    # just in case, checking if the value is not None
+                    if message is not None:
+                        if self.logger.level <= logging.DEBUG:
+                            # The following logging prints every single received message
+                            # except empty message data ones.
+                            type = WSMsgType(message.type)
+                            message_type = (
+                                type.name if type is not None else message.type
                             )
+                            message_data = message.data
+                            if isinstance(message_data, bytes):
+                                message_data = message_data.decode("utf-8")
+                            if len(message_data) > 0:
+                                # To skip the empty message that Slack server-side often sends
+                                self.logger.debug(
+                                    f"Received message "
+                                    f"(type: {message_type}, "
+                                    f"data: {message_data}, "
+                                    f"extra: {message.extra}, "
+                                    f"session: {session_id})"
+                                )
 
-                    # The logging here is for detailed trouble shooting of potential issues in this client.
-                    # If you don't see this log for a while, it means that
-                    # this receive_messages execution is no longer working for some reason.
-                    if self.logger.level <= logging.DEBUG:
-                        counter_for_logging = (
-                            counter_for_logging + 1
-                        ) % logging_interval
-                        if counter_for_logging == 0:
-                            log_message = (
-                                f"{logging_interval} WebSocket messages received "
-                                f"after the previous same log ({session_id})"
-                            )
-                            self.logger.debug(log_message)
+                            counter_for_logging = (
+                                counter_for_logging + 1
+                            ) % logging_interval
+                            # The logging here is for detailed trouble shooting of potential issues in this client.
+                            # If you don't see this log for a while, it can mean that
+                            # this receive_messages execution is no longer working for some reason.
+                            if self.trace_enabled and counter_for_logging == 0:
+                                log_message = (
+                                    f"{logging_interval} WebSocket messages received "
```

Comment:
Similarly here, if my understanding of this message is correct, I would probably phrase it like what I suggested above:

    Checkpoint: WebSocket message received (count={counter_for_logging} session={session_id})

Reply:
I would like to avoid accumulating the number to avoid overflow errors in ultimate situations (say, a very busy app running for years without restart), just in case. Perhaps, we can say like this instead:

> The receive_messages method has been working without any issues (session: {session_id}, logging interval: {logging_interval})

Reply:
OK. I'd suggest removing the leading "The" at beginning for consistency with the earlier, similar message.